src/index.js

```diff
@@ -0,0 +1,70 @@
+import * as task from './status';
+import addEvents from './drag&drop'; // eslint-disable-line
+import './style.css';
+
+let list = [
+];
+
+function listIt() {
+  if (window.localStorage.getItem('todos')) {
+    const todos = window.localStorage.getItem('todos');
+    list = JSON.parse(todos);
+  }
+  document.querySelector('.list').innerHTML = '';
+  list.forEach((item) => {
+    const taskElement = document.createElement('li');
+    taskElement.classList.add('task', 'draggable');
+    if (item.isCompleted) {
+      taskElement.classList.add('completed');
+    }
+    const delBtn = document.createElement('button');
+    const delIcon = document.createElement('i');
+    delIcon.classList = 'far fa-trash-alt icon del';
+    delBtn.appendChild(delIcon);
+    delBtn.classList.add('del-container');
+    delBtn.addEventListener('click', () => {
+      task.removeThis(item, list);
+      listIt();
+    });
+    const checker = document.createElement('input');
+    checker.type = 'checkbox';
+    checker.classList.add('task-check');
+    checker.addEventListener('click', () => {
+      task.toggle(item, list);
+      listIt();
+    });
+    checker.checked = item.isCompleted;
+    const taskText = document.createElement('input');
+    taskText.classList = 'task-text';
+    taskText.value = item.description;
+    taskText.addEventListener('change', () => {
+      if (taskText.value.length > 0) {
+        item.description = taskText.value;
+        task.saveLocal(list);
+      }
+    });
+    const dragIcon = document.createElement('i');
+    dragIcon.classList = 'fas fa-ellipsis-v drag icon';
+    taskElement.appendChild(delBtn);
+    taskElement.appendChild(checker);
+    taskElement.appendChild(taskText);
+    taskElement.appendChild(dragIcon);
+    taskElement.draggable = 'true';
+    document.querySelector('.list').appendChild(taskElement);
+  });
+  document.querySelector('.input-task').focus();
+  addEvents(list);
+}
```

Comment:
this function seems to be completing multiple tasks at once, what do you think of dividing the main task into several sub-tasks and distributing them among respective functions for instance: 
we can create two separate functions to delete completed tasks and and add new tasks:

`const add = () => {
......do add 
}
const delete = () =>{
......do deletin
}

const listit = () =>{
condition invoke add()
condition invoke delete()
}`

since you are invoking list function inside its own, it can be considered as a sub-function so it would be best to make the lists function do one single task: actual listing thing and create the integrating function which will call each unit to make the whole app work as expected 

Making functions do only one task actually makes your code more clean, solid and easy to maintain.